src/main/groovy/io/jenkins/infra/repository_permissions_updater/ArtifactoryPermissionsUpdater.groovy

```diff
@@ -178,13 +101,13 @@ public class ArtifactoryPermissionsUpdater {
                     ]
                 }.flatten().join(',')
                 excludesPattern ''
-                repositories([ 'releases', 'snapshots' ])
+                repositories([ 'snapshots' ]) // TODO switch back to 'releases' to be effective
```

Comment:
Just for testing purposes I guess?

Reply:
Yes, will be changed before merge, I just didn't want to accidentally grant everything to everyone or something. I don't exactly have a second set of Artifactories and Githubs around, so development is against the live system 🤷